### PR TITLE
MSEDO-275 - Edit Active Substances

### PR DESCRIPTION
**Pull request by andrenns**

### CODE REVIEW

File: src/main/webapp/app/entities/active-substance/list/active-substance.component.html [hunk 1]
```diff
@@ -1,10 +1,10 @@
 <div class="mainWrapper">
   <div class="listHeaderWrapper">
     <h2 class="h2-color" id="page-heading" data-cy="ActiveSubstanceHeading" customTranslate="supportivecareApp.activeSubstance.home.title">Active Substances</h2>
-    <button *customHasAnyAuthority="'ROLE_USER'" id="jh-create-entity" data-cy="entityCreateButton" class="btn btn-primary jh-create-entity button-create" [routerLink]="['/active-substance/new']">
+    <!--<button *customHasAnyAuthority="'ROLE_USER'" id="jh-create-entity" data-cy="entityCreateButton" class="btn btn-primary jh-create-entity button-create" [routerLink]="['/active-substance/new']">
```
Comment (aamarques) [inline src/main/webapp/app/entities/active-substance/list/active-substance.component.html:4]: Remove Comments from code before submitting it to Master

File: src/main/webapp/app/entities/active-substance/list/active-substance.component.html [hunk 2]
```diff
@@ -1,10 +1,10 @@
...
       <fa-icon icon="pills" slot="start"></fa-icon>
       <span customTranslate="supportivecareApp.activeSubstance.home.createLabel"> Create a new Active Substance </span>
-    </button>
+    </button>-->
```
Comment (aamarques) [inline src/main/webapp/app/entities/active-substance/list/active-substance.component.html:7]: Remove Comments from code before submitting it to Master

File: src/test/javascript/cypress/integration/entity/active-substance.spec.ts [hunk 3]
```diff
@@ -60,7 +60,7 @@ describe.only('ActiveSubstance e2e test', () => {
   //   cy.visit('/');
   // });
 
-  it('should load create ActiveSubstance page', () => {
+  /*it('should load create ActiveSubstance page', () => {
```
Comment (aamarques) [inline src/test/javascript/cypress/integration/entity/active-substance.spec.ts:63]: Remove Comments from code before submitting it to Master

File: src/test/javascript/cypress/integration/entity/active-substance.spec.ts [hunk 4]
```diff
@@ -70,7 +70,7 @@ describe.only('ActiveSubstance e2e test', () => {
     cy.getEntityCreateHeading('ActiveSubstance');
     cy.get(entityCreateSaveButtonSelector).should('exist');
     cy.visit('/');
-  });
+  });*/
```
Comment (aamarques) [inline src/test/javascript/cypress/integration/entity/active-substance.spec.ts:73]: Remove Comments from code before submitting it to Master

**Review by aamarques — CHANGES_REQUESTED** [review]
Is it a good approach to verify/remove all variables that will be never used as showed on annotatios

**Comment by andrenns** [comment]
> Is it a good approach to verify/remove all variables that will be never used as showed on annotatios

The tests are commented because when QA decides they will be corrected and used

And the button is commented because PO was not sure if that button should be deleted or not, so if in a sprint or two she decides we should use it is there, otherwise we will remove it

**Comment by aamarques** [comment]
Merged to master

**Pull request merged**